system/datatypes/peer_reviewed_article.php

```diff
@@ -1182,10 +1182,13 @@ private function getCommentFormHtml($type, DocPage $discussion = null) {
 			$nameFieldName = self::FIELD_NAME_DISCUSSION_COMMENTER_NAME;
 			$email = __('art-email');
 			$emailFieldName = self::FIELD_NAME_DISCUSSION_COMMENTER_EMAIL;
+			$unanonymous =  __('art-comment-unanonymous');
 			$html .= <<<EOF
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<strong><label for="$nameFieldName"> $name </label></strong> <input type="text" id="$nameFieldName" name="$nameFieldName" />
+				<strong><label for="$nameFieldName"> $name $unanonymous </label>  </strong><input type="text" id="$nameFieldName" name="$nameFieldName" />
```

Comment:
I wonder if this unanonymous remark should be in the `<label>` tag, or perhaps outside of it? I wonder if it is conceptually part of the label or not? Or should this perhaps be an info-icon?